packages/contracts-bedrock/test/CommonTest.t.sol

```diff
@@ -169,58 +170,113 @@ contract L2OutputOracle_Initializer is CommonTest {
     }
 }
 
-contract Portal_Initializer is L2OutputOracle_Initializer {
-    // Test target
-    OptimismPortal internal opImpl;
-    OptimismPortal internal op;
-    SystemConfig systemConfig;
+contract SuperchainConfig_Initializer is L2OutputOracle_Initializer {
```

Comment:
This diff is hard to read, what I did is:
1. add a new `SuperchainConfig_Initializer` contract
2. move `SystemConfig_Initializer` to this file rather than in `SystemConfig.t.sol`
3. Put both of those before `Portal_Initializer` because it depends on them being deployed already.